pygmt/__init__.py

```diff
@@ -1,11 +1,17 @@
 # pylint: disable=missing-docstring
 #
-# The main API for PyGMT.
+# PyGMT is a library for processing geospatial and geophysical data and making
+# publication quality maps and figures. It provides a Pythonic interface for
+# the Generic Mapping Tools (GMT), a command-line program widely used in the
+# Earth Sciences.
 #
-# All of PyGMT is operated on a "modern mode session" (new to GMT6). When you
-# import the pygmt library, a new session will be started automatically. The
-# session will be closed when the current Python process terminates. Thus, the
-# Python API does not expose the `gmt begin` and `gmt end` commands.
+# Main Features
+# -------------
+# Here are just a few of the things that PyGMT does well:
+#   - Easy handling of all kinds of geospatial data including gridded and
+#     raster data.
+#   - Allows plotting of a large spectrum of objects on maps including lines,
```

Comment:
How about removing "on maps" or revising it to be "on figures"? or we may consider a map and a figure are the same thing?

Reply:
Good point, modified that part.